spec/support/components/work_packages/settings_menu.rb

```diff
@@ -56,7 +56,7 @@ def open!
       end
 
       def expect_open
-        expect(page).to have_selector(selector)
+        page.find(selector)
```

Comment:
The change itself is fine. But can we please rename the method then. The `expect_*` are currently used to have expectations in them. Of sorts, the find is also an expectation as it will raise if the element is not found but I would still have expected something different.